daemon/restart.go

```diff
@@ -30,9 +31,9 @@ func (daemon *Daemon) ContainerRestart(ctx context.Context, name string, options
 // container. When stopping, wait for the given duration in seconds to
 // gracefully stop, before forcefully terminating the container. If
 // given a negative duration, wait forever for a graceful stop.
-func (daemon *Daemon) containerRestart(ctx context.Context, daemonCfg *configStore, container *container.Container, options containertypes.StopOptions) error {
+func (daemon *Daemon) containerRestart(ctx context.Context, daemonCfg *configStore, cnt *container.Container, options containertypes.StopOptions) error {
```

Comment:
quick comment / nit (haven't looked at your latest changes); perhaps use `ctr` as name for the variable; I know we use that in other places.

Reply:
This [commit](https://github.com/moby/moby/pull/45695/commits/4c0c7472394563e8ad5cc8255034fc9523eb24d9) renames `cnt` to `ctr`